### DISCUSSION TITLE
365 score not updating

### DISCUSSION THREAD

**Discussion by IPsteve** [mondoohq/cnquery#3575] [General]
I worry we are doing something wrong, our 365 score is showing some items not completed when they are.  Am I missing something?

![image](https://github.com/mondoohq/cnquery/assets/163515908/48268ae9-419e-4787-85be-5abf4e09f394)

![image](https://github.com/mondoohq/cnquery/assets/163515908/fc27b808-9e44-44af-a71a-950b27adb02b)

**Comment by HRouhani** [comment]
Thank you for bringing this matter to our attention again. Given that this issue has been reported previously, it seems there might be an underlying inconsistency within the MS365 cloud systems. To investigate this further and provide accurate assistance, could you please establish a terminal connection to your MS365 infrastructure? We would like you to execute a specific query to gather more details. The query we need you to run is:

```
microsoft.domains {passwordValidityPeriodInDays}
``` 

Please execute this query in your terminal connected to the MS365 infrastructure and share the results with us. This information will be crucial for identifying and resolving the discrepancy you're experiencing.

An example for you here:
![Screenshot from 2024-03-15 10-31-25](https://github.com/mondoohq/cnquery/assets/56231339/bcb85d0b-0bba-4bd5-9186-08c518568c86)
